index.html

```diff
@@ -0,0 +1,197 @@
+        </li>
+        <li>
+          <img src="img/peace icon.svg">
+        </li>
+      </ul>
+    </section><br><br><br><br><br><br><br><br>
+  </div>
+
+
+  <div>
+    <section class="together">
+      <img src="img/Snapshoot Portfolio.svg">
+      <div class="textp">
+        <h2>Tonic</h2>
+        <p class="inline-paragraph" id="canopy">CANOPY</p>
+        <img src="img/dot.svg">
+        <p class="inline-paragraph" id="devend">Back End Dev</p>
+        <img src="img/dot.svg">
+        <p class="inline-paragraph" id="unique">2023</p>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section class="multipost">
+      <img src="img/multipost.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+
+
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/tonic2.svg" class="postsvg">
+      <h2>Tonic</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/multi2.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
```

Comment:
- [ ] The `<section>` element is a structural HTML element used to group together related elements. following the best practice, Your entire `work section` is meant to be wrapped in a `section tag`, then individual project cards can be a `div` and not a section, I'll suggest you remove the sections tag you have around each individual project card and have only one section tag wrapping the whole project section. :+1: :100: :smile: